discovery-provider/src/solana/solana_client_manager.py

```diff
@@ -48,7 +50,7 @@ def handle_get_sol_tx_info(client, index):
                         f"solana_client_manager.py | get_sol_tx_info | Fetching tx {tx_sig} {endpoint}"
                     )
                     tx_info: ConfirmedTransaction = client.get_confirmed_transaction(
-                        tx_sig
+                        tx_sig, encoding
```

Comment:
should we set a default here

Reply:
it has been set https://github.com/AudiusProject/audius-protocol/pull/2815/files/6294cce955b3c17dc8c8536483697d5f34b098d2#diff-39a8aeea9ddb7f0ada929e3065acd259bdcf2fb08f853068d61b3394d4616a3dR40